sdk/src/androidTest/java/com/mapbox/maps/RenderCacheTest.kt

```diff
@@ -7,6 +7,7 @@ import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeoutException
 
+@MapboxExperimental
```

Comment:
Is this line still needed?

Reply:
yes, it is required to avoid warning messages, as the render cache is still experimental feature.